chall01/spentti.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+import sys
+import re
+
+morse = {
+	'a': '.-',
+	'b': '-...',
+	'c': '-.-.',
+	'd': '-..',
+	'e': '.',
+	'f': '..-.',
+	'g': '--.',
+	'h': '....',
+	'i': '..',
+	'j': '.---',
+	'k': '-.-',
+	'l': '.-..',
+	'm': '--',
+	'n': '-.',
+	'o': '---',
+	'p': '.--.',
+	'q': '--.-',
+	'r': '.-.',
+	's': '...',
+	't': '-',
+	'u': '..-',
+	'v': '...-',
+	'w': '.--',
+	'x': '-..-',
+	'y': '-.--',
+	'z': '--..',
+	' ': ' '
+}
+
+if (len(sys.argv) == 2):
```

Comment:
to avoid your code to be too much indented; I would recommend going the other way
```python
if len(sys.argv) != 2:
  print_usage()
  exit(1)

word = sys.argv[1].lower.strip()
```